examples/conda-env-kernel-image/Dockerfile

```diff
@@ -1,4 +1,4 @@
 FROM continuumio/miniconda3:4.9.2
 
 COPY environment.yml .
-RUN conda env create -f environment.yml
+RUN conda env update -f environment.yml --prune
```

Comment:
sry missed this earlier. Why do we need this change? what is prune trying to achieve?

Reply:
In the original sample, this iwll create a new conda environment called `myenv`. Then, since `base` environment is being selected by default in Studio, so how do you switch to `myenv` conda?
With this change, I override the `base` environment, which is selected by default by Studio.